comps/dataprep/redis/multimodal_langchain/requirements.txt

```diff
@@ -0,0 +1,20 @@
+docarray[full]
+fastapi
+langchain==0.1.12
+langchain_benchmarks
+langsmith
```

Comment:
please remove langsmith

Reply:
Thank for your feedback. We have removed langsmith here.